tests/omp/test_omp_transeq.f90

```diff
@@ -47,7 +47,7 @@ program test_omp_transeq
   L_global = [2*pi, 2*pi, 2*pi]
 
   ! Domain decomposition in each direction
-  nproc_dir = [nproc, 1, 1]
+  nproc_dir = [1, 1, nproc]
```

Comment:
Any particular reason for this change? Should we consider running variations?

Reply:
Yes, this is because 2decomp&fft only allows 2D decomposition and as it is implemented at the moment it is in y and z. I think that's also a limitation of the cuda backend, but I am not sure anymore. 
That could likely be changed in the future though, whether this should be part of this PR or not I am not sure.